docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1084 @@
+# Voice Control Manager
+Voice control manager features allow the user to record voice and give responses for recognized voice commands. You can register general and system voice commands such as "power on", "power off", "music play", "music stop", and so on. In addition, you can start and stop voice recording. When recording voice is finished, you can receive multiple recognition results such as Automatic Speech Recognition (ASR) result and matched commands among commands list which is registered by application using voice control client.
```

Comment:
Suggested rephrase:

Voice control manager features allow you to record voice and give responses for the recognized voice commands....